samples/corda/fungible-house-token/extension-flows/AssetTransferHelperFlows.kt

```diff
@@ -0,0 +1,246 @@
+package net.corda.samples.tokenizedhouse.flows
```

Comment:
Similar comments as in Simple AssetFlows file.

Reply:
Done.